router/transformer/transformer.go

```diff
@@ -241,6 +245,7 @@ func (trans *handle) Transform(transformType string, transformMessage *types.Tra
 	}
 	func() { httputil.CloseResponse(resp) }()
 
+	destinationJobs.Hydrate(jobs)
```

Comment:
Why do we need to hydrate the jobs twice?

Reply:
we don't